docs/source/sim_overview.rst

```diff
@@ -56,55 +54,28 @@ give you at least a basic exploratory idea of the performance of your review:
+
+To make your simulations reproducible you can use the ``--seed`` and ``--init_seed`` options. 'Init
+seed' controls the starting set of papers to train the model on, while the 'seed' controls the
+random generator after initialization.
 
-To make your simulations reproducible you can use the ``--seed`` and ``--init_seed`` options. Init
-seed controls the starting set of papers to train the model on. While the normal seed controls the
-random generator after initialization. For initialization you can also use ``--prior_idx`` to set
-the papers to start the simulation with.
+By default, the model initializes with one relevant and one irrelevant record. If you want to initialize your model with a specific set of starting papers, you can use ``--prior_idx`` to select the indices of the papers you want to start the simulation with.
 
-To limit the size of your state file and the time to simulate, you can increase ``--n_instances``,
-which reduces the number of training iterations.
+The ``--n_instances`` argument controls the number of records that have to be labelled before the model is retrained again and is set at 1 by default. If you want to reduce the number of training iterations, for example to limit the size of your state file and the time to simulate, you can increase ``--n_instances``.
 
-Use a different classifier with the ``-m`` flag. Names for implemented models are listed on the
-`models <models.html>`__ page.
+You can select a classifier with the ``-m`` flag, which is set to be Naive Bayes by default. Names for implemented classifiers are listed on the :doc:`models <models.html>`page.
 
 Implemented query strategies are listed on the `query strategy <query_strategies.html>`__ page and
-is set with the ``-q`` option.
+can be set with the ``-q`` option.
 
-For feature feature extraction, supply the ``-e`` flag, with more details on the page for 
+For feature extraction, supply the ``-e`` flag. Default is TF-IDF, more details are on the page for
 `feature extraction <feature_extraction.html>`__.
 
-The last model that can be changed is the `balance strategy <balance_strategies.html>`__, and is
-changed with the ``-b`` flag.
-
-Finally, options can also be supplied through a configuration file with ``--config_file``.
-For example:
-
-.. code-block:: ini
-
-	[global_settings]
-	n_instances = 1
-	n_prior_included = 0
-	n_prior_excluded = 0
-	model = nb
-	balance_strategy = double
-	query_strategy = max
-	feature_extraction = tfidf
-	
-	[balance_param]
-	a = 0.7492527339524988
-	alpha = 1.376755934059147
-	b = 0.15013204593328155
-	
-	[feature_param]
-	ngram_max = 2
-	split_ta = 0
-	use_keywords = 0
-	
-	[model_param]
-	alpha = 0.8267010267867003
+The last element that can be changed is the `balance strategy <balance_strategies.html>`__, and is
+changed with the ``-b`` flag. Default is triple balance.
```

Comment:
We should change that...

Reply:
I've created a PR #316 updating the default query strategy to max and balance strategy to double